metadata-transformer/src/test/java/no/unit/nva/metadata/service/MetadataServiceTest.java

```diff
@@ -46,35 +54,113 @@
     public static final String DC_IDENTIFIER = "DC.identifier";
     public static final String DC_LANGUAGE = "DC.language";
     public static final String CITATION_TITLE = "citation_title";
+    public static final String DATE_SEPARATOR = "-";
+    public static final String ARTICLE_HTML = "article.html";
+    public static final String YEAR_ONLY = "2001";
+    public static final String FULL_DATE = "2001-12-19";
+    public static final String DC_DATE = "dc.date";
+    public static final String VALID_DATE = "2017";
 
     private WireMockServer wireMockServer;
 
+    @AfterEach
+    void tearDown() {
+        wireMockServer.stop();
+    }
+
     @ParameterizedTest(name = "#{index} - {0}")
```

Comment:
Is this line still needed if you're specifying names individually in the tests? Perhaps do so for all the tests and remove this line.